backend/tools/test.go

```diff
@@ -69,10 +70,10 @@ const (
 						VALUES ($1, $2, $3, '', $4, $5, $6, $7, $8)`
 
 	getRequest = `SELECT * FROM request WHERE id = $1`
-  
-	insertOffer = `INSERT INTO offer (name, description, extrainfo, minprice, maxprice, userid, categoryid, timestamp) 
-						VALUES ($1, $2, '', $3, $4, $5, $6, $7)`
-	getOffer = `SELECT * FROM offer WHERE (name, description, userid, categoryid) = ($1, $2, $3, $4)`
+
+	insertOffer = `INSERT INTO offer (id, name, description, extrainfo, minprice, maxprice, userid, categoryid, timestamp) 
+						VALUES ($1, $2, '', $3, $4, $5, $6, $7, $8)`
```

Comment:
You might want to use the order ($1, $2, $3, '',, $4, $5, $6, $7, $8) to you avoid putting an offer with an empty description in the DB. I had the same problem while dealing with Requests and it'll be useful when you'll implement the Search.

Reply:
Resolved in commit 0ab3231